invertedai/api/blame.py

```diff
@@ -48,6 +48,41 @@ def blame(
     """
     Parameters
     ----------
+    location:
```

Comment:
the variable uses `get_birdviews` but in the docstring is `get_birdview`; and `detect_collisions`  is missing in the docstring

Reply:
Because `detect_collisions` is only for internal usage, I skipped it here, so it won't occur in the public doc.